/**:
  ros__parameters:
    nav_distance_tol: 0.35
    nav_orientation_tol: 0.35
    action_in_paused: False # Flag to command action when navigation paused
    use_sim_time: False

